packages/jsii-pacmak/lib/targets/golang/types/interface.ts

```diff
@@ -3,61 +3,121 @@ import { InterfaceType, Method, Property } from 'jsii-reflect';
 import { GoType } from './go-type';
 import { GoTypeRef } from './go-type-reference';
 import { Package } from '../package';
+import { TypeField } from './type-field';
+import { getFieldDependencies } from '../util';
+
+class InterfaceProperty implements TypeField {
```

Comment:
Since this go type should only refer to a behavioral interface, there shouldn't be any properties on this, correct?

Reply:
Behavioral interfaces have properties and methods. DataType interfaces have only properties. We could merge the logic of these two. This class only serves to replace the inline emit of property getters/setters in `Interface` so that we can resolve dependencies of each one.